dashboard/.stylelintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": ["stylelint-config-recommended-scss", "stylelint-config-prettier"]
```

Comment:
stylelint-config-recommended-scss is what stylelint recommends to use when using SCSS + prettier. However, this only enables the "possible errors" rules listed in https://github.com/stylelint/stylelint/blob/master/docs/user-guide/rules.md#possible-errors and none of the language feature rules. These are pretty basic rules, so I'm not sure if we'd want some of the more specific rules?